content/_changelogs/10.7.0.md

```diff
@@ -0,0 +1,81 @@
+  [#22798](https://github.com/cypress-io/cypress/issues/22798), and
+  [#22935](https://github.com/cypress-io/cypress/issues/22935)
+- Vue.js projects with a custom spec pattern configuration can now use the
+  "Create Spec from Component" capability introduced in v10.5.0. Addresses
+  [#23071](https://github.com/cypress-io/cypress/issues/23071)
+- Angular component testing projects can provide a custom configuration to the
+  Webpack dev server using the `projectConfig` key. This enables support for
+  projects that don't adhere to Angular CLI conventions. Addresses
+  [#23161](https://github.com/cypress-io/cypress/issues/23161)
+- The Cypress `clock` can now be set to a new time using the `setSystemTime`
+  function. Addresses
+  [#15424](https://github.com/cypress-io/cypress/issues/15424)
+
+**Performance:**
+
+- The `cy.session()` command's behavior has been enhanced to no longer clear the
+  dom after a validation function runs. This means a `cy.visit()` command is no
+  longer required after running `cy.session()` when a validation function is
+  used. Addresses [#22368](https://github.com/cypress-io/cypress/issues/22368).
+
+**Bugfixes:**
+
+- Fixed an issue where Cypress could crash if a websocket upgrade request was
+  made over HTTPS. Fixes
+  [#22217](https://github.com/cypress-io/cypress/issues/22217)
+- Fixed an issue where filtering with the `--spec` run mode option would not
+  find any specs if the project was located at the root directory of a file
+  system. Fixes [#23380](https://github.com/cypress-io/cypress/issues/23380)
+- Fixed an issue where projects using Angular v14.2.0 and above would be unable
+  to run component tests. Fixes
+  [#23585](https://github.com/cypress-io/cypress/issues/23585)
+- Fixed an issue where snapshots performed in XHR requests could reflect the
+  primary domain instead of current domain. Fixes
+  [#21496](https://github.com/cypress-io/cypress/issues/21496)
+- Introduced a new `GITHUB_RUN_ATTEMPT` environment variable when claiming specs
+  from Cypress Cloud to address a potential scenario that would cause invalid
+  Github Action requests to be accepted. Addressed in
+  [#23445](https://github.com/cypress-io/cypress/pull/23445)
+- Fixed an issue where an internal TypeScript type was exposed globally. Fixes
+  [#23333](https://github.com/cypress-io/cypress/pull/23333)
+- Fixed an issue where an incomplete `devServer` configuration would cause the
+  specs list to not display in the Cypress App. Fixes
+  [#23486](https://github.com/cypress-io/cypress/issues/23486)
+
+**Miscellaneous:**
+
+- Display a default icon in the browser selection menu for browsers without
+  bundled icons. Fixes
+  [#21766](https://github.com/cypress-io/cypress/issues/21766)
+
+**Dependency Updates:**
+
+- Upgraded the bundled TypeScript version shipped with Cypress from `4.2.3` to
+  `4.7.4`. Addressed in
+  [#22252](https://github.com/cypress-io/cypress/pull/22252).
```

Comment:
```suggestion
  [#23523](https://github.com/cypress-io/cypress/pull/23523).
```

Reply:
@ZachJW34 I feel like I'm misremembering, was this bump to enable typescript & node 18 support?

Reply:
No it was to use some new TS features internally, not related to node 18

Reply:
Thanks Zach. This sounds internal-only so I'll remove it